test/test_rdict.py

```diff
@@ -242,6 +242,9 @@ def test_delete_range(self):
 
     def test_reopen(self):
         self.test_dict.close()
+
+        self.assertRaises(DbClosedError, self.test_dict.get("b"))
+
```

Comment:
Looks like this should be: 
```python
self.assertRaises(DbClosedError, lambda: self.test_dict.get("b"))
```

Currently this line throws error and fails the test.

https://stackoverflow.com/a/6103930